docs/faq.txt

```diff
@@ -132,4 +132,17 @@ Determined has support for TensorFlow models that use the :ref:`tf.keras <tf-ker
 or :ref:`Estimator <estimator-trial>` APIs. For models that use the low-level TensorFlow
 Core APIs, we recommend porting your model to use :ref:`Estimator Trial <estimator-trial>`.
 `Example of converting a Tensorflow graph into an Estimator
-<https://github.com/determined-ai/determined/blob/master/examples/experimental/mnist_tp_to_estimator/model_def.py>`_.
+<https://github.com/determined-ai/determined/blob/master/examples/experimental/trial/mnist_tp_to_estimator/model_def.py>`_.
+
+
+Tensorboard Support
+-------------------
+
+Can I log Tensorboards events in Determined?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Yes you can log TF Events to ``/tmp/tensorboard``. For more details see the
+:ref:`Tensorboard How-To Guide <data-in-tensorboard>`.
+
+Can I use Tensorboard with PyTorch?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Yes! For an example of this check out the `mnist-GAN example <https://github.com/determined-ai/determined/blob/master/examples/official/trial/mnist_gan_pytorch/model_def.py>`_.
```

Comment:
Would you mind also updating the TensorBoard How-To guide with this information? A little more detail would be nice, as well -- i.e., is using `determined.tensorboard.metric_writers.pytorch.TorchWriter` necessary? (If so, maybe we should add that to the API reference -- but we can do that later.)

Reply:
Added a bit more detail here, also added info about PyTorch to the How-To guide and a reference guide for `determined.tensorboard.metric_writers.pytorch.TorchWriter`.